setup.py

```diff
@@ -0,0 +1,76 @@
+from io import open
+from os.path import abspath, dirname, join
+
+from setuptools import setup, find_packages
+
+here = abspath(dirname(__file__))
+
+with open(join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+with open(join(here, 'requirements.txt'), encoding='utf-8') as f:
+    requires = f.read().splitlines()
+
+setup(
+    name='moler',  # Required
+    version='0.0.4',  # Required
+    description='Moler is library to help in building automated tests',  # Required
+    long_description=long_description,  # Optional
+    long_description_content_type='text/markdown',  # Optional (see note above)
+    url='https://github.com/nokia/moler',  # Optional
+    author='Nokia',  # Optional
+    license='BSD 3-Clause',
+    classifiers=[  # Optional
+        'Development Status :: 3 - Alpha',
+
+        'Environment :: Console',
+
+        'Intended Audience :: Developers',
+        'Intended Audience :: System Administrators',
+        'Intended Audience :: Telecommunications Industry',
+        'Topic :: Software Development :: Build Tools',
+
+        'License :: OSI Approved :: BSD License',
+
+        'Operating System :: Microsoft :: Windows',
```

Comment:
In current version some features (like Terminal, Devices) do not work under Windows.